frontend/tests/gqlHelpers.ts

```diff
@@ -0,0 +1,19 @@
+import { expect, type APIRequestContext } from '@playwright/test';
+import { serverBaseUrl } from './envVars';
+
+export function validateGqlErrors(json: {error: unknown, data: unknown}, expectError = false): void {
+  if (!expectError) {
+    expect(json.error).toBeFalsy();
+    expect(json.data).toBeDefined();
+    // TODO: Dive into data object properties and make sure none of them have any "errors" subproperties
```

Comment:
This I think?
```suggestion
    Object.values(json.data).forEach(value => expect(value.errors).toBeFalsy());
```

Reply:
Thanks!

Reply:
Done in commit 0e3162c0541f34f97a0d616d043df3a29060e1cb.